### ISSUE TITLE
colorizeModelLine does not work if you try to use immediately after creating the model

### ISSUE THREAD

**Issue by davicatunda** [microsoft/monaco-editor#2678]
The code bellow using `monaco.editor.createModel` and  `monaco.editor.colorizeModelLine` results into a non-colorized line, but the second time we run (on this case with a setTimeout) it works correctly.

**monaco-editor version:** 0.28
**Browser:** Chrome
**OS:** Windows
**Playground code that reproduces the issue:**
On https://microsoft.github.io/monaco-editor/playground.html try
```js
const code = `/* Some example CSS */`
const model = monaco.editor.createModel(code, 'css');
const element = document.getElementById('root');
element.innerHTML = monaco.editor.colorizeModelLine(model, 1);
window.setTimeout(() => {
    element.innerHTML = monaco.editor.colorizeModelLine(model, 1);
}, 1000)
```

**Comment by alexdima** [comment]
Yes, the only ones that work really well are `colorizeElement` or `colorize` as they return a promise when the language has finished loading. The `colorizeModelLine` one will colorize the model's line using the current state of things (i.e. as if the file would be displayed in an editor).
